frontend/tsconfig.json

```diff
@@ -11,6 +12,9 @@
     "types": [
       "vite/client"
     ],
+    "paths": {
+      "~/*": ["src/*"]
+    },
```

Comment:
A path alias which allows to no longer us relative paths which will break if you move the file around.